packages/sampler/stories/default/DropDown.js

```diff
@@ -0,0 +1,34 @@
+import DropDown, {DropDownBase} from '@enact/moonstone/DropDown';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {action} from '@storybook/addon-actions';
+import {withInfo} from '@storybook/addon-info';
+
+import {boolean, select, text} from '../../src/enact-knobs';
+import {mergeComponentMetadata} from '../../src/utils';
+
+const Config = mergeComponentMetadata('DropDown', DropDown, DropDownBase);
+DropDown.displayName = 'DropDown';
```

Comment:
I believe this works better if it is _before_ the `Config` line. We should just update every story all at once and be done with it. #soemothertime